docs/aws-sdk-for-iOS-with-minio.md

```diff
@@ -59,5 +61,6 @@ for example if you running that eample project
 1. Run the xcode project on your phone or simulator
 2. Click upload button on screen
 <img src="/docs/screenshots/iOS-test-app.png" alt="screenshot" height="250">
-
-3. Check on `Minio Browser`, inside images bucket, there should be an image there called test.jpg, means you susccessful upload the image
+3. Set bucket policy prefix as `prefix` (becasue as you see the example code, remote name set as prefix_test.jpg) from `Minio Browser` or `Minio mc` to `images` bucket
```

Comment:
It is not clear what is meant here, could you please rephrase to make it clearer?